src/caffe/util/io.cpp

```diff
@@ -32,35 +33,35 @@ using google::protobuf::io::CodedOutputStream;
 using google::protobuf::Message;
 
 bool ReadProtoFromTextFile(const char* filename, Message* proto) {
-  int fd = open(filename, O_RDONLY);
-  CHECK_NE(fd, -1) << "File not found: " << filename;
-  FileInputStream* input = new FileInputStream(fd);
+  FILE* fp = fopen(filename, "r");
+  CHECK_NOTNULL(fp);
```

Comment:
This should give a more user-friendly error message.  People will file issues over this.